Dockerfiles/dogstatsd/alpine/Dockerfile

```diff
@@ -16,4 +15,4 @@ HEALTHCHECK --interval=1m --timeout=5s --retries=2 \
   CMD ["/probe.sh"]
 
 ENTRYPOINT ["/entrypoint.sh"]
-CMD ["/dogstatsd", "start"]
+CMD ["/dogstatsd", "-c", "/", "start"]
```

Comment:
Noob question: any chance people are overriding the dogstatsd.yaml file in the old location ?

Reply:
Good point, I'm moving the default file to `/etc/datadog-agent/dogstatsd.yaml` instead